bookshelf/api_v1.py

```diff
@@ -151,6 +151,18 @@ def arch():
     return result
 
 
+def disable_openssh_rdns(distribution):
+    log_green('Disabling openssh reverse dns lookups')
+    openssh_config_file = '/etc/ssh/sshd_config'
+    dns_config = 'UseDNS no'
+    if not file_contains(openssh_config_file, dns_config, use_sudo=True):
+        file_append(openssh_config_file, dns_config, use_sudo=True)
+        service_name = 'sshd'
+        if 'ubuntu' in distribution:
+            service_name = 'ssh'
```

Comment:
As I understand it the purpose of this library is to abstract those details
away from the caller, so I used a pattern I saw elsewhere in the file.

However I'm not sure that it's 100% consistent across the functions.

I could split this in to two functions, one to do the core work, the other
to decide which service to restart which calls the first, but it doesn't
seem like a big win here.

I'll wait to see what @Azulinho says.